portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,31 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.List; 
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+@WebServlet("/welcome-data")
 public class DataServlet extends HttpServlet {
+  private List<String> welcomeTexts;
+
+  @Override
+  public void init(){
+      welcomeTexts = new ArrayList();
+      welcomeTexts.add("Welcome");
+      welcomeTexts.add("Isten hozott!");
+      welcomeTexts.add("Bine ai venit!");
+      welcomeTexts.add("Willkommen!");
+  }
+
+  String getRandomWelcomeText(){
```

Comment:
Nit: this could be private. We tend to minimize the visibility of methods and classes as much as possible. 

go/java-practices/visibility#minimize

go/ej3e-15

Reply:
Fixed
(I don't have access to the first link.)